README.md

```diff
@@ -10,21 +10,33 @@ Set up a virtualenv (assuming you already installed virtualenv with pip) and ins
 
 and `deactivate` to leave the virtualenv.
 
+## Services keys
+You need a copy of `./firebase.local.json` and `./keys.local.json`. You can
+copy the templates and add your own keys or get the keys from Mozilla's
+engineering gdrive. If you don't have access, please contact a team member.
+
 ## Firebase
-Add the path to the Firebase credentials json file to `config.py`, and then run
+With the keys ready, run:
 
     python -m samples.firebase-test
 
 You can view real-time changes to the database on the console.
 
-## Testing
+## Point to production DB in local builds
+Useful when you're running pre-caching or logging scripts.
+
+In [app/constants.py][const-tableprefix], change to:
```

Comment:
This is intentional, that prox-server should not point to production/. For virtually any action that you're taking, it should make changes to your local my-user/ table.

I think that if you want to add a line specifically about pre-caching to production, I would do that at the bottom, very clearly under "pre-caching to production" possibly with a warning that any scripts you run will be writing directly to production. Basically, we should take care to not pollute production, and suggesting this as the "first" item to change seems ill-advised.